streaming/python/tests/test_word_count.py

```diff
@@ -45,14 +46,17 @@ def sink_func(x):
                 (old_value[0], old_value[1] + new_value[1])) \
         .sink(sink_func)
     ctx.submit("word_count")
-    import time
-    time.sleep(3)
+
+    def check_succeed():
+        if os.path.exists(sink_file):
+            with open(sink_file, "r") as f:
```

Comment:
Do we need to deal with IO error here?

Reply:
If file exists, open and file shouldn't fail.